src/button/borderless.scss

```diff
@@ -18,55 +19,50 @@
   }
 
   // If icon is by itself, button takes 1:1 size ratio
-  > .iui-icon:only-child {
+  > .iui-button-icon:only-child {
     margin-left: $iui-component-offset;
     margin-right: $iui-component-offset;
   }
 
-  &:enabled:hover,
-  &:enabled:active,
+  &:hover,
   &:focus-visible {
     @include themed {
-      background-color: rgba(
-        t(iui-color-foreground-body-rgb),
-        t(iui-opacity-6)
-      );
+      background-color: rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-6));
       border-color: transparent;
     }
   }
 
-  &.iui-active:not(.iui-high-visibility):not(.iui-cta) {
-    &:enabled,
-    &:enabled:hover,
-    &:enabled:active {
+  &.iui-active {
+    &,
+    &:hover {
       background: transparent;
       @include themed {
-        background-color: rgba(
-          t(iui-color-foreground-primary-rgb),
-          t(iui-opacity-6)
-        );
+        background-color: rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-6));
       }
       border-color: transparent;
+
+      > .iui-label {
```

Comment:
I'm thinking iui-label also needs to be iui-button-label

Reply:
Changed to iui-button-label